interview-questions.md

```diff
@@ -1,57 +1,89 @@
+<!-- First, without external resources. Challenge yourself to answer from memory as if you were in a job interview. If you don't know the answer a good tip is to answer a simpler version of the question. -->
 
-Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
+<!-- Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS. -->
 
-INSTRUCTOR EXAMPLE: What is a conditional statement?
+<!-- INSTRUCTOR EXAMPLE: What is a conditional statement? -->
 
-Your answer:
+<!-- Your answer: A conditional statement is a statement that compares different values or inputs. It evaluates a condition to determine whether to perform the code in it or not. It has to meet a certain true or false criteria in the condition. Only one `if`, one `else`, and as many `else if` as desired. -->
 
-Researched answer:
+<!-- Researched answer: Conditional statements help your code make decisions based on true conditions. -->
+<!-- For example,  -->
+  
+  <!-- if(value1.length > value2.length){
+    <!-- return `${value1} has more characters than ${value2}` -->
+  <!-- } else if(value1.length === value2.length) { -->
+    <!-- return `${value1} has the same amount of characters as ${value2}` -->
+  <!-- } else { -->
+      <!-- return `${value2} has more characters than ${value1}` -->
+  <!-- } -->
 
-1. What is git? What is the difference between git and Github?
+1. What is git? What is the difference between git and Github? 
 
-Your answer:
+Your answer: Git is a version control software whereas, Github is an online platform that uses Git technology. The difference between the two is Github allows you to manage many different projects on a server compared to git allows us to access our repos on our local.
 
-Researched answer:
+Researched answer: Git is a version control tool (installed locally) that Github (hosted in the cloud) is built on top of, we do not need Github to use Git. Github allows its user to store repo's from our local's onto the remote as a backupup of our work, allows its users to share as well as access other user's repositories. Whereas, Git is on our local and allows its user's to update our current project, this saves the process in versions. With these versions it permits the users to see the updates of their current project throughout.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Relational operators evaluate the relationship between two numbers and will return a boolean(true or flase).
 
-Researched answer:
+For example:
+console.log(5 > 1)
+<!-- //output: true-->
+
```

Comment:
I love that you are giving a lot of examples! It's a really good sign because you are elaborating. Elaborating in an interview is excellent.